examples/jupyter-src/Export-BouncingBall.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
```

Comment:
Please add at the very top (highly visible) that this is just a placeholder example and will be replaced by something better soon :-)

Reply:
done